<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>skinClusterProxyMeshQWindow</class>
 <widget class="QWidget" name="skinClusterProxyMeshQWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="proxySettingQGroup">
     <property name="title">
      <string>Settings:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="proxySettingsQLayout">
        <item>
         <widget class="QLabel" name="proxySuffixQLabel">
          <property name="text">
           <string>Sufix:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="proxySuffixQLine">
          <property name="text">
           <string>SkinProxy</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="weightClampQLabel">
          <property name="text">
           <string>Weight Clamp:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="weightClampQSpin">
          <property name="suffix">
           <string/>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.025000000000000</double>
          </property>
          <property name="value">
           <double>0.100000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="proxyParentQLayout">
        <item>
         <widget class="QLabel" name="proxyParentQLabel">
          <property name="text">
           <string>Proxy Parent:</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="proxyParentOptionsQLayout">
          <item>
           <widget class="QRadioButton" name="proxyParentSourceQRadio">
            <property name="text">
             <string>Source Mesh Parent</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="proxyParentInfluenceQRadio">
            <property name="text">
             <string>Influence</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="proxyParentWorldQRadio">
            <property name="text">
             <string>World</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="proxyParentCustomQRadio">
            <property name="text">
             <string>Custom</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="proxyParentCustomQLayout">
          <item>
           <widget class="QLineEdit" name="proxyParentCustomQLine">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="proxyParentCustomQButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>&lt;&lt;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="toolModeQGroup">
     <property name="title">
      <string>Tool Mode:</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QRadioButton" name="meshesModeQRadio">
        <property name="text">
         <string>Meshes Mode</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="influenceModeQRadio">
        <property name="text">
         <string>Influence Mode</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="modeQLayout"/>
   </item>
   <item>
    <spacer name="modeQSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="applyQButton">
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="applyQProgress">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
